app/models/policy_event.rb

```diff
@@ -12,7 +12,7 @@ def self.create_events(target, event, result)
     event = MiqEventDefinition.find_by(:name => event) if event.kind_of?(String)
     chain_id = nil
     result.each do |r|
-      miq_policy_id = r[:miq_policy].kind_of?(MiqPolicy) ? r[:miq_policy].id : nil # handle built-in policies too
+      miq_policy_id = r[:miq_policy].id if r[:miq_policy].kind_of?(MiqPolicy) # handle built-in policies too
```

Comment:
similar to my statement above about explicitly settings a var to nil in a loop versus not changing the variable.

Reply:
yea, as mentioned before. If it is a variable declared in a loop, it gets `null` on every iteration.
For me, this version is easier to read. So I'd like to keep it this new way.
But willing to revert if you wish.

Reply:
This is fine :+1: